### PR TITLE
Drtii 894 experiment with machine learning by adding it

### PR DESCRIPTION
**Pull request by richbirch**
- Add new job to train touchdown models
- Upgrade Akka to 2.6
- Upgrade drt-lib to 148
- Tests to run sequentially to avoid race conditions with Spark Sessions
- Remove duplicated code that is now included in drt-lib

### CODE REVIEW

File: src/main/scala/uk/gov/homeoffice/drt/analytics/services/PassengerCounts.scala [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import uk.gov.homeoffice.drt.analytics.time.SDate
+import uk.gov.homeoffice.drt.ports.AirportConfig
+
+import scala.concurrent.{ExecutionContext, Future}
+
+
+object PassengerCounts {
+  private val log = LoggerFactory.getLogger(getClass)
+
+  def apply(config: AirportConfig, daysToLookBack: Int)
```
Comment (ngu04) [inline src/main/scala/uk/gov/homeoffice/drt/analytics/services/PassengerCounts.scala]: renaming to **update** sound better here

File: src/test/scala/uk/gov/homeoffice/drt/analytics/prediction/ArrivalTimeSpec.scala [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      implicit system =>
+        implicit ec =>
+          implicit mat =>
+            val eventualResult = TouchdownTrainer.train(150, 20, T1)
+            val result: Seq[Option[Double]] = Await.result(eventualResult, 5.minutes)
+            val total = result.size
+            val modelCount = result.count(_.isDefined)
+            val threshold = 10
+            val improvements = result.collect { case Some(imp) if imp >= threshold => imp }.size
+            println(s"total: $total, models: $modelCount, $improvements >= $threshold%")
```
Comment (ngu04) [inline src/test/scala/uk/gov/homeoffice/drt/analytics/prediction/ArrivalTimeSpec.scala]: println here

**Review by ngu04 — APPROVED** [review]
🥇

File: src/main/scala/uk/gov/homeoffice/drt/analytics/services/PassengerCounts.scala (hunk 1 above)
Comment (richbirch) [reply src/main/scala/uk/gov/homeoffice/drt/analytics/services/PassengerCounts.scala]: Good call!

File: src/test/scala/uk/gov/homeoffice/drt/analytics/prediction/ArrivalTimeSpec.scala (hunk 2 above)
Comment (richbirch) [reply src/test/scala/uk/gov/homeoffice/drt/analytics/prediction/ArrivalTimeSpec.scala]: :+1:

**Pull request merged**